{{ cookiecutter.repo_name }}/setup.py

```diff
@@ -4,7 +4,8 @@
     name='src',
     packages=find_packages(),
     version='0.1.0',
+    install_requires=['{% if cookiecutter.include_starter_proj == "Y" %}scikit-learn[alldeps]{% endif %}'],
```

Comment:
I think this is overly complex - is it enough to just put `scikit-learn` in the requirements.txt?

Also there's a bug here in that the `if` is inside the quotes so you'll end up with `install_requires=['']` if the conditional isn't met.